interpreter/exec/simd.ml

```diff
@@ -11,6 +11,7 @@ let lanes shape =
   | F32x4 -> 4
   | F64x2 -> 2
 
+let i8x16_indices = [0; 1; 2; 3; 4; 5; 6; 7; 8; 9; 10; 11; 12; 13; 14; 15]
 let i16x8_indices = [0; 2; 4; 6; 8; 10; 12; 14]
 let i32x4_indices = [0; 4; 8; 12]
 let f32x4_indices = i32x4_indices
```

Comment:
You shouldn't need these definitions if you simply use List.init instead of List.map. See example suggestions below.

Reply:
Sg, replaced these indices with use of `List.init`.